<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data_process</class>
 <widget class="QMainWindow" name="Data_process">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data_process</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="Output">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>801</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="csv_path">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="csv_choose">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Replace_outliers">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>替换异常值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入检测列：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Checkcol">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入检测等级：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Checkl">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowFin">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>跳过：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="jmpRow">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>缺失值标识：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Missing">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Check_Fix_Miss">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>缺失值检测修补</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_Miss">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>缺失值删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>重复判定依据：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Duplicate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Check_Duplicate">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>重复值检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_Duplicate">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>重复值删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Sign_edit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>检测列：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Check_miss_col">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>检测列：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Check_duplicate_col">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="csv_out">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>输出CSV</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>作为自变量的列：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Independent_variable">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>118</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>作为因变量的列：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Dependent_variable">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Regression">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>输出线性回归方程</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
